src/ILLink.RoslynAnalyzer/COMAnalyzer.cs

```diff
@@ -88,11 +88,11 @@ static bool IsComInterop (ISymbol symbol)
 				if (typeSymbol == null)
 					return false;
 
-				if (typeSymbol.ContainingNamespace.Name == "System" && typeSymbol.Name == "Array") {
+				if (typeSymbol.ContainingNamespace?.Name == "System" && typeSymbol.Name == "Array") {
```

Comment:
Don't we have enabled nullable to catch such errors?

Reply:
See https://github.com/dotnet/linker/issues/2687.